boards/shields/ci_shield/Kconfig.defconfig

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2020 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_CI_SHIELD
+
+config SPI
+	default y
+
+config DISK_ACCESS_SPI_SDHC
+	default y
+
+config DISK_ACCESS_SDHC
+	default y
```

Comment:
We want application to control activation, so it should be:
```
if DISK_ACCESS_SDHC

config SPI
	default y

config DISK_ACCESS_SPI_SDHC
	default y

endif
```

Reply:
The application (prj.conf) is configuring CONFIG_DISK_ACCESS=y
So could the `boards/shields/ci_shield/Kconfig.defconfig`  configure with depends on DISK_ACCESS